Unable to resolve @nrwl/react:application when following the guide "[React] Migrating existing code bases"

## Current Behavior
I'm attempting to create a proof of concept for migrating an existing React application (custom build configuration, not create-react-app) by following this migration guide https://nx.dev/latest/react/migration/overview - However, I'm unable to proceed after creating an empty workspace. 

```
➜  rat-nx-migration git:(master) ✗ npx create-nx-workspace@latest
npx: installed 192 in 46.321s
? Workspace name (e.g., org name)     post-migration
? What to create in the new workspace empty             [an empty workspace with a layout that works best for building apps]
? Use Nx Cloud? (It's free and doesn't require registration.) No
Creating a sandbox with Nx...
added 192 packages from 155 contributors in 9.964s

23 packages are looking for funding
  run `npm fund` for details

new rat-nx-migration --no-interactive --preset=empty --appName= --linter=eslint --no-nxCloud --collection=@nrwl/workspace
✔ Packages installed successfully.
    Directory is already under version control. Skipping initialization of git.
CREATE rat-nx-migration/nx.json (552 bytes)
CREATE rat-nx-migration/tsconfig.base.json (467 bytes)
CREATE rat-nx-migration/README.md (3263 bytes)
CREATE rat-nx-migration/.editorconfig (245 bytes)
CREATE rat-nx-migration/.gitignore (503 bytes)
CREATE rat-nx-migration/.prettierignore (74 bytes)
CREATE rat-nx-migration/.prettierrc (26 bytes)
CREATE rat-nx-migration/workspace.json (97 bytes)
CREATE rat-nx-migration/package.json (1169 bytes)
CREATE rat-nx-migration/tools/tsconfig.tools.json (223 bytes)
CREATE rat-nx-migration/tools/generators/.gitkeep (0 bytes)
CREATE rat-nx-migration/.vscode/extensions.json (109 bytes)
CREATE rat-nx-migration/apps/.gitkeep (0 bytes)
CREATE rat-nx-migration/libs/.gitkeep (0 bytes)

➜  rat-nx-migration git:(master) ✗ cd post-migration
➜  post-migration git:(master) ✗ nx generate @nrwl/react:application rat-app
Unable to resolve @nrwl/react:application.
Cannot find module '@nrwl/react/package.json'
Require stack:
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/src/shared/workspace.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/src/commands/generate.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/index.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/cli/lib/init-local.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/cli/bin/nx.js
- /Users/atay/.nvm/versions/node/v12.16.2/lib/node_modules/nx/node_modules/@nrwl/cli/bin/nx.js
- /Users/atay/.nvm/versions/node/v12.16.2/lib/node_modules/nx/bin/nx.js
```

## Expected Behavior
Should be able to run `nx generate @nrwl/react:application my-application` without any errors or able to complete the steps listed in the migration guide https://nx.dev/latest/react/migration/overview.

## Steps to Reproduce
```
- npx create-nx-workspace@latest
-- empty workspace

- nx generate @nrwl/react:application my-application
```

### Failure Logs
```
nx generate @nrwl/react:application rat-app
Unable to resolve @nrwl/react:application.
Cannot find module '@nrwl/react/package.json'
Require stack:
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/src/shared/workspace.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/src/commands/generate.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/tao/index.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/cli/lib/init-local.js
- /Users/atay/Dev/nx-playground/rat-nx-migration/post-migration/node_modules/@nrwl/cli/bin/nx.js
- /Users/atay/.nvm/versions/node/v12.16.2/lib/node_modules/nx/node_modules/@nrwl/cli/bin/nx.js
- /Users/atay/.nvm/versions/node/v12.16.2/lib/node_modules/nx/bin/nx.js
```

### Environment
```
  Node : 12.16.2
  OS   : darwin x64
  npm  : 6.14.4
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.18
  @nrwl/cypress : Not Found
  @nrwl/devkit : 11.0.18
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : Not Found
  @nrwl/linter : Not Found
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.18
  @nrwl/web : Not Found
  @nrwl/workspace : 11.0.18
  typescript : 4.0.5
```

--- a/docs/shared/migration/overview.md
+++ b/docs/shared/migration/overview.md
@@ -34,10 +34,22 @@ The `apps` directory is the place where your top-level applications will be stor
 For Angular applications:
 
 ```bash
+npm install --save-dev @nrwl/angular
+```
+
+and
+
+```bash
 nx generate @nrwl/angular:application my-application
 ```
 
 For React applications:
+
+```bash
+npm install --save-dev @nrwl/react
+```
+
+and
 
 ```bash
 nx generate @nrwl/react:application my-application

